site/gdocs/OwidGdocPage.tsx

```diff
@@ -317,28 +320,34 @@ export default function OwidGdocPage({
                 ></script>
             </Head>
             <body>
-                <SiteHeader
-                    isOnHomepage={gdoc.content.type === OwidGdocType.Homepage}
-                    archiveInfo={isOnArchivalPage ? archiveContext : undefined}
-                />
-                <div id="owid-document-root">
-                    <AriaAnnouncerProvider>
-                        <DebugProvider debug={debug}>
-                            <OwidGdoc
-                                {...gdoc}
-                                isPreviewing={isPreviewing}
-                                archiveContext={archiveContext}
-                            />
-                        </DebugProvider>
-                        <AriaAnnouncer />
-                    </AriaAnnouncerProvider>
-                </div>
-                <SiteFooter
-                    context={SiteFooterContext.gdocsDocument}
-                    debug={debug}
-                    isPreviewing={isPreviewing}
-                    archiveContext={archiveContext}
-                />
+                <QueryClientProvider client={queryClient}>
```

Comment:
Do we even need this one, when `hydrateOwidGdoc` also provides a query client? And is there an adverse effect to a React tree being wrapped with two (different) `QueryClientProvider`s?